beacon_node/network/src/network_beacon_processor/rpc_methods.rs

```diff
@@ -437,12 +437,12 @@ impl<T: BeaconChainTypes> NetworkBeaconProcessor<T> {
                     }
                 }
                 Err(e) => {
-                    // TODO(das): lower log level when feature is stabilized
-                    error!(
+                    // The node is expected to be able to serve these columns, but it fails to retrieve them.
+                    warn!(
                         block_root = ?data_column_ids_by_root.block_root,
                         %peer_id,
                         error = ?e,
-                        "Error getting data column"
+                        "Error getting data column for by root request "
```

Comment:
Small nit: the "for" here is a typo?
```suggestion
                        "Error getting data column by root request "
```

Reply:
ah it actually meant "Error getting data column for DataColumnByRoot request", but i thought that was too verbose.

Reply:
oh, that makes sense now, looks good